weblate/templates/translate.html

```diff
@@ -676,6 +676,18 @@ <h5 class="list-group-item-heading">
                 </div>
               {% endif %}
 
+              {% if unit.automatically_translated %}
+                <div class="list-group-item check check-item">
+                  <h5>
+                    <span class="green">{% icon "check.svg" %}</span>
+                    {% translate "Automatically translated" %}
+                  </h5>
+                  <p class="list-group-item-text check-description">
+                    {% blocktranslate with author=unit.get_last_author %}This string was translated automatically by {{ author }}.{% endblocktranslate %}
```

Comment:
Adding a dismiss button (which would really just save the string without changes) here would make it obvious how to get rid of this alert.